docs/tasks/administer-cluster/cilium-network-policy.md

```diff
@@ -0,0 +1,76 @@
+---
+assignees:
+- danwent
+title: Use Cilium for NetworkPolicy
+---
+
+{% capture overview %}
+This page shows how to use Cilium for NetworkPolicy.
+
+For background on Cilium, read the [Introduction to Cilium](http://cilium.readthedocs.io/en/latest/intro/).
+{% endcapture %}
+
+{% capture prerequisites %}
+* Install Cilium for Kubernetes.
```

Comment:
Perhaps the prerequisite should simply be to have a Kubernetes cluster up and running. You can replace this line with

`* {% include task-tutorial-prereqs.md %}`

That will include the generic requirements.